packages/cli/src/commands/proxy/connect.ts

```diff
@@ -0,0 +1,127 @@
+import { ux, Args, Flags } from '@oclif/core'
+import { set } from 'lodash'
+import { tunnelServerFlags, urlFlags } from '@preevy/cli-common'
+import { commands, profileStore, withSpinner, SshConnection } from '@preevy/core'
+import { inspect } from 'util'
+import { formatPublicKey } from '@preevy/common'
+import { spawn } from 'child_process'
+import { connectToTunnelServerSsh } from '../../tunnel-server-client'
+import ProfileCommand from '../../profile-command'
+import { filterUrls, printUrls } from '../urls'
+
+// eslint-disable-next-line no-use-before-define
+export default class Connect extends ProfileCommand<typeof Connect> {
+  static description = 'Tunnel existing local compose application'
+
+  static flags = {
+    ...tunnelServerFlags,
+    ...urlFlags,
+    ...ux.table.flags(),
+    'private-env': Flags.boolean({
```

Comment:
I think private-env isn't that clear. Maybe something like "--access=private", WDYT?